### ISSUE TITLE
Translation

### ISSUE THREAD

**Issue by michailkouroupakis** [alibaba-fusion/cool#1]
Could you please make accessible website and  plugin for people who can read English. It will be awesome :D . Thanks really good work!

**Comment by wwsun** [comment]
yes, we are considering the English version, it will arrive soon.
